modules/images-triggering-updates-imagestream-changes-kubernetes-about.adoc

```diff
@@ -12,25 +12,29 @@ The annotation is defined as follows:
 
 [source,yaml]
 ----
-Key: image.openshift.io/triggers
-Value:
-[
- {
-   "from": {
-     "kind": "ImageStreamTag", <1>
-     "name": "example:latest", <2>
-     "namespace": "myapp" <3>
-   },
-   "fieldPath": "spec.template.spec.containers[?(@.name==\"web\")].image", <4>
-   "paused": false <5>
- },
- ...
-]
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    image.openshift.io/triggers:
+      [
+       {
+         "from": {
+           "kind": "ImageStreamTag", <1>
+           "name": "example:latest", <2>
+           "namespace": "myapp" <3>
+         },
+         "fieldPath": "spec.template.spec.containers[?(@.name==\"web\")].image", <4>
+         "paused": false <5>
+       },
+      # ...
+      ]
+# ...
 ----
 <1> Required: `kind` is the resource to trigger from must be `ImageStreamTag`.
 <2> Required: `name` must be the name of an image stream tag.
 <3> Optional: `namespace` defaults to the namespace of the object.
-<4> Required: `fieldPath` is the JSON path to change. This field is limited and accepts only a JSON path expression that precisely matches a container by ID or index. For pods, the JSON path is  "spec.containers[?(@.name='web')].image".
+<4> Required: `fieldPath` is the JSON path to change. This field is limited and accepts only a JSON path expression that precisely matches a container by ID or index. For pods, the JSON path is  `spec.containers[?(@.name='web')].image`.
```

Comment:
There are two spaces between: "path is" and "`spec.containers[?(@.name='web')]."